emulator/src/main/java/com/scalar/client/tool/emulator/EmulatorTerminal.java

```diff
@@ -208,10 +207,13 @@ private boolean parseAndRunCommand(String line) {
       if (line.startsWith(command.getCommandName())) {
         String params = line.replaceFirst(command.getCommandName(), "").trim();
         String[] paramsArray = params.isEmpty() ? new String[] {} : paramsParser(params);
-        command.parseWithHandlers(
-            new CommandLine.RunFirst(),
-            new CommandExceptionHandler(terminal.getTerminal().writer()),
-            paramsArray);
+        try {
+          command.parseWithHandler(
```

Comment:
The format is correct ?

Reply:
Thank you for detailed review.
I think the format is correct